sdk/typespec/typespec_client_core/src/async_runtime/tests.rs

```diff
@@ -241,14 +241,29 @@ fn test_get_runtime() {
     let _runtime = get_async_runtime();
 }
 
+struct DummyRuntime;
```

Comment:
Nit: I remember reading somewhere we shouldn't use words like "dummy". Maybe just "TestRuntime" or something?

Reply:
Fixing, you're right.